tests/unit/services/shieldPersistenceService-retention.test.js

```
@@ -0,0 +1,536 @@
+    not: jest.fn().mockReturnThis(),
+    lte: jest.fn().mockReturnThis(),
+    gte: jest.fn().mockReturnThis(),
+    limit: jest.fn().mockReturnThis(),
+    order: jest.fn().mockReturnThis(),
+    range: jest.fn().mockReturnThis(),
+    single: jest.fn().mockReturnThis(),
+    __setResult: function(result) {
+      this.__result = result;
+      return this;
+    },
+    then: jest.fn().mockImplementation(function(resolve) {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)